docs/source/hyperparameters.rst

```diff
@@ -160,39 +164,29 @@ for. The user would have to search the file to find what is parametrized.
         def train_dataloader(self):
             return DataLoader(mnist_train, batch_size=self.hparams.batch_size)
 
-One way to get around this is to convert a Namespace or dict into key-value pairs using `**`
-
-.. code-block:: python
-
-    parser = ArgumentParser()
-    parser = LitMNIST.add_model_specific_args(parser)
-    args = parser.parse_args()
-    dict_args = vars(args)
-    model = LitMNIST(**dict_args)
-
-Within any LightningModule all the arguments you pass into your `__init__` will be stored in
-the checkpoint so that you know all the values that went into creating this model.
-
-We will also add all of those values to the TensorBoard hparams tab (unless it's an object which
-we won't). We also will store those values into checkpoints for you which you can use to init your
-models.
+4. You can also save full objects such as `dict` or `Namespace` to the checkpoint.
 
 .. code-block:: python
 
+    # using a argparse.Namespace
     class LitMNIST(LightningModule):
 
-        def __init__(self, layer_1_dim, some_other_param):
+        def __init__(self, conf, *args, **kwargs):
             super().__init__()
-            self.layer_1_dim = layer_1_dim
-            self.some_other_param = some_other_param
+            self.hparams = conf
 
-            self.layer_1 = torch.nn.Linear(28 * 28, self.layer_1_dim)
+            # equivalent
+            self.save_hyperparameters(conf)
 
-            self.layer_2 = torch.nn.Linear(self.layer_1_dim, self.some_other_param)
-            self.layer_3 = torch.nn.Linear(self.some_other_param, 10)
+            self.layer_1 = torch.nn.Linear(28 * 28, self.hparams.layer_1_dim)
+            self.layer_2 = torch.nn.Linear(self.hparams.layer_1_dim, self.hparams.layer_2_dim)
+            self.layer_3 = torch.nn.Linear(self.hparams.layer_2_dim, 10)
 
+    conf = OmegaConf(...)
```

Comment:
This part is half baked.

Maybe say something about accepting OmegaConf config objects without specifying how they are created.
This will be meaningful for both people using OmegaConf directly and people using Hydra.

If you want to show how to instantiate, use 

[OmegaConf.create()](https://omegaconf.readthedocs.io/en/latest/usage.html#creating)

(With a link to the docs).

Reply:
mind draft a follow-up PR? :rabbit:

Reply:
Not at this time, I am extremely busy with the Hydra 1.0 release.
Maybe just change it to:
```suggestion
    conf = OmegaConf.create(...)
```